File: c/gpt/p18.c
// p18: input -> value at index or -1
#include <stdio.h>

int getValue(int idx)
{
    int arr[] = {2, 4, 6, 8, 10, 12};
    int n = sizeof(arr) / sizeof(arr[0]);
    if (idx < 0 || idx >= n)
        return -1;
    return arr[idx];
}

int main(void)
{
    int idx;
    printf("Index: ");
    if (scanf("%d", &idx) != 1)
    {
        printf("-1\n");
        return 0;
    }
    printf("%d\n", getValue(idx));
    return 0;
}

File: javascript/gpt/p18.js
// p18: input -> value at index or -1
const readline = require("readline");

function getValue(idx) {
  const arr = [2, 4, 6, 8, 10, 12];
  if (idx < 0 || idx >= arr.length) return -1;
  return arr[idx];
}

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});
rl.question("Index: ", (ans) => {
  const idx = parseInt(ans.trim(), 10);
  if (Number.isNaN(idx)) console.log(-1);
  else console.log(getValue(idx));
  rl.close();
});
